resources/assets/lib/models/chat/channel.ts

```diff
@@ -133,16 +138,13 @@ export default class Channel {
   }
 
   @action
-  updateMessage(message: Message) {
-    const messageObject = _.find(this.messages, {uuid: message.uuid});
-    if (messageObject) {
-      messageObject.update(message);
-      if (messageObject.errored) {
-        messageObject.messageId = messageObject.uuid; // prevent from being culled by uniq sort thing
-      } else {
-        messageObject.persist();
-      }
+  updateMessage(message: Message, json: MessageJSON | null) {
```

Comment:
((again) probably not this line but just so it can be marked resolved)

Message order seems a bit weird. First message is sent and then it moves up (last message should be "what's with the order")

![](https://s.myconan.net/2020-10/firefox_2020-10-16_20-05-45.png)

Reply:
probably something to do with server and client time slightly off and the message not updated with actual timestamp anymore